Solutions/Azure Active Directory/Analytic Rules/AzureAADPowerShellAnomaly.yaml

```diff
@@ -44,10 +44,10 @@ entityMappings:
   - entityType: Account
     fieldMappings:
       - identifier: FullName
-        columnName: AccountCustomEntity
+        columnName: UserPrincipalName
```

Comment:
Fullname is not a strong entity identifier for Account (its actually a legacy mapping) where we have a UPN we should split in into Name and UPN suffix and have both as entities - https://learn.microsoft.com/en-us/azure/sentinel/entities-reference

Reply:
Also for these we should make the AAD User ID an Account entity